Src/EB/AMReX_EB_utils.cpp

```diff
@@ -196,11 +196,23 @@ facets_nearest_pt (IntVect const& ind_pt, IntVect const& ind_loop, RealVect cons
 }
 }
 
-void FillSignedDistance (MultiFab& mf, EB2::Level const& ls_lev,
-                         EBFArrayBoxFactory const& eb_factory, int refratio,
+void FillSignedDistance (MultiFab& mf_in, EB2::Level const& ls_lev,
+                         EBFArrayBoxFactory const& eb_factory_in, int refratio,
                          bool fluid_has_positive_sign)
 {
-    AMREX_ALWAYS_ASSERT(mf.is_nodal());
+    AMREX_ALWAYS_ASSERT(mf_in.is_nodal());
+
+    // because the algorithm below is N^2 in the number of points per box,
+    // we always do this operation with a max grid size of 32.
+    auto new_ba = mf_in.boxArray();
+    new_ba.maxSize(32);
+    auto new_dm = DistributionMapping(new_ba);
+    int max_guard = eb_factory_in.getBndryCent().nGrow();
+    auto eb_factory_ptr = amrex::makeEBFabFactory(ls_lev.Geom(), new_ba, new_dm,
+                                                  {max_guard, max_guard, max_guard},
+                                                  amrex::EBSupport::full);
+    EBFArrayBoxFactory const& eb_factory = *eb_factory_ptr;
+    MultiFab mf(new_ba, new_dm, mf_in.nComp(), mf_in.nGrow());
```

Comment:
Another issue. We need to use `nGrowVect()`.

Reply:
The tiling way seems to work, so we don't this MultiFab anyway

Reply:
That's great! Could you update the PR description so that we can use that as the commit message.